app/assets/stylesheets/helpers/_header.scss

```diff
@@ -284,5 +284,36 @@
         }
       }
     }
+
+    input {
+      @include core-14;
+
+      display: block;
+      margin: 0 20px 0 20px;
+      max-width: 940px;
+      width: 100%;
+
+      @include ie-lte(6) {
+        width: 940px;
+      }
+    }
+
+    label {
+      @include core-14;
+      margin: 0 0 0 20px;
+    }
+
+    div.actions {
```

Comment:
I don't think we need to over qualify this.

https://github.com/CSSLint/csslint/wiki/Disallow-overqualified-elements